doc/developer/debugging.rst

```diff
@@ -57,4 +61,35 @@ account:
 
     sudo /etc/init.d/postgres reload
 
-Logs are available in the ``/var/log/postgresql/postgresql-9.0-main.log`` file.
+Logs are available in the ``/var/log/postgresql/postgresql-9.*-main.log`` file.
+
+Performance or network error
+----------------------------
+
+For performance and proxy issue be sure that add the internal URL in the config file
```

Comment:
issue => issues
be sure that add the internal URL in the config file are on localhost => make sure that all internal URLs in the config file use localhost